oauth2_http/java/com/google/auth/oauth2/DefaultCredentialsProvider.java

```diff
@@ -80,9 +80,16 @@
    * Returns the Application Default Credentials.
    *
    * <p>Returns the Application Default Credentials which are credentials that identify and
-   * authorize the whole application. This is the built-in service account if running on Google
-   * Compute Engine or credentials specified by an environment variable or a file in a well-known
-   * location.</p>
+   * authorize the whole application. Application Default Credentials are looked for in the
+   * following order:
```

Comment:
The wording would probably be better:

"The following locations are used in the order shown to look for Application Default Credentials:"